.github/workflows/documentation-links.yml

```diff
@@ -0,0 +1,21 @@
+name: Read the Docs PR preview
+on:
+  pull_request_target:
+    types:
+      - opened
+    paths:
+    - 'Doc/**'
+    - 'Misc/**'
```

Comment:
Almost every PR adds a news entry, and we have a few dozens of them every day. Should we create such a workload for Netlify? It (probably) wastes build queue slots (that harms us) and storage space (that harms the Nethify team).

```suggestion
```

Reply:
Err, do you mean the RTD and the RTD team? This PR is specifically replacing Netlify with RTD. Also, this job doesn't actually run RTD, it just adds the RTD preview link to the PR OP, which is a comparatively cheap operation.

That being said, still not sure if its worth adding the link just for NEWS changes; it is helpful to not have to build the docs to check them, but considering the hassle of finding the specific changelog entry I highly doubt it will be used enough to make sense. Also, the existing docs build and check job still runs on such PRs anyway (as it should as otherwise it would be easy to inject reST syntax errors in NEWS entries), so we still get coverage from that.